classes/Combination.php

```diff
@@ -71,6 +71,12 @@ class CombinationCore extends ObjectModel
     /** @var bool Low stock mail alert activated */
     public $low_stock_alert = false;
 
+    /**
+     * @deprecated since 1.7.8
+     * @see StockAvailable::$quantity instead
+     *
+     * @var int
```

Comment:
```suggestion
     * @var int|null
```

Reply:
It cannot be null. Quantity is loaded from stock in constructor